paasta_itests/steps/setup_marathon_job_steps.py

```diff
@@ -93,6 +96,29 @@ def create_complete_app(context):
         assert 'deployed' in return_tuple[1]
 
 
+@when(u'we create a complete app with {number} instance(s)')
+def create_complete_app_with_instances(context, number):
+    change_number_of_context_instances(context, number)
+    create_complete_app(context)
+
+
+@when(u'we change the number of instances to {number}')
+def change_number_of_context_instances(context, number):
+    fake_service_config['instances'] = int(number)
+
+
+@when(u'we run setup_marathon_job until the instance count is {number}')
+def run_setup_marathon_job(context, number):
+    app_instances = -1
+    while app_instances != int(number):
+        try:
+            app_instances = context.marathon_client.get_app(fake_appid).instances
```

Comment:
I'm surprised that this is called "instances" here, I thought it was called "tasks"?